examples/config-ldap.yaml

```diff
@@ -11,7 +11,7 @@ connectors:
   name: OpenLDAP
   id: ldap
   config:
-    host: localhost:10389
+    host: localhost:389
```

Comment:
If there's a strong opinion to use the nonstandard port, I can revert it.
But then the port would be different here (outside the container) and when using "docker exec" (inside the container).

Reply:
I think the rationale behind using a non-standard port is avoiding potential collision with an existing service. But all the other examples (mysql, etc) use standard ports, so it should be okay.